digdag-core/src/main/java/io/digdag/core/schedule/ScheduleExecutor.java

```diff
@@ -199,8 +199,13 @@ private void runSchedule(ScheduleControl lockedSched)
 
             Config scheduleConfig = SchedulerManager.getScheduleConfig(def);
             boolean skipOnOvertime = scheduleConfig.get("skip_on_overtime", boolean.class, false);
+            boolean disableBackfill = scheduleConfig.get("disable_backfill", boolean.class, false);
 
-            if (!activeAttempts.isEmpty() && skipOnOvertime) {
+            if (disableBackfill && now.isAfter(sched.getNextRunTime())) {
```

Comment:
This `sched` comes from `io.digdag.core.database.DatabaseScheduleStoreManager#lockReadySchedules` and the method selects schedules which have older next run time than current time https://github.com/treasure-data/digdag/blob/8cf4af37be92602595bb6b10a41ef1e5961f3fb2/digdag-core/src/main/java/io/digdag/core/database/DatabaseScheduleStoreManager.java#L253

So it seems you don't need to check scheduler's next run time here.

Reply:
No. This condition is important part.  where clause you mentioned is "after equal". the code is "after". it's means the only when just "next_run_time" is allowed to run.

Reply:
I found the condition is too strict.
new code allows one second delay to run.